BedrockServer.cpp

```diff
@@ -1862,3 +1914,87 @@ void BedrockServer::_acceptSockets() {
     }
 }
 
+void BedrockServer::_loadSecureData(const map<BedrockPlugin*, list<string>>& pluginSecureDataMap, const SData& args) {
+    // Do we need to load secure data? Skip loading if we don't. In this case, plugins
+    // should manage loading debug secure data inside of the plugin.
+    if (!args.isSet("-loadSecureData")) {
```

Comment:
Why even have this flag? Plugins can not ask for this data when `-debug` is set, right?

Reply:
Because we call `plugin->needsSecureData()` before we call `plugin->initialize()`, the plugins don't have a reference to args yet, so they don't know what flags are or are not set. I could pass a reference to `needsSecureData()` for args if you think that's better.

Reply:
Did we decide to do the thing in your previous comment?